src/zhinst/toolkit/command_table.py

```diff
@@ -220,11 +220,16 @@ class HeaderEntry(ParentEntry):
     Args:
         schema: JSON schema of the node.
         path: Path representation of the node.
+        version: JSON schema version
     """
 
-    def __init__(self, schema: dict, path: tuple):
+    def __init__(self, schema: dict, path: tuple, version: str = ""):
```

Comment:
```suggestion
    def __init__(self, schema: dict, path: tuple, version: str = None):
```

Does anything speak against this?